book/website/reproducible-research/ci/ci-coverage.md

```diff
@@ -0,0 +1,2 @@
+## Adding Code Coverage to your CI Workflow
```

Comment:
I think this section should be moved to the "Code testing" chapter as a specific type of test. For this CI chapter then we could mention the different actions/integrations that use a coverage service to render the result of the coverage runs. (e.g. codecov, coveralls, but there are more listed in GH's marketplace).